src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -1,8 +1,9 @@
 /* eslint-disable no-underscore-dangle */
 import * as React from 'react';
 import {StyleSheet, Text, View} from 'react-native';
-import {useNavigation} from '@react-navigation/native';
+import {useNavigation, useRoute} from '@react-navigation/native';
 
+import PropTypes from 'prop-types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useRoute` hook is imported but it's not used in the old code. It's good to see that it's now being utilized in the new code.
```diff
- import {useNavigation} from '@react-navigation/native';
+ import {useNavigation, useRoute} from '@react-navigation/native';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->